src/components/ExpandedViewMode/ExpandedViewModeContent/ActivityHistory/ActivityHistory.js

```diff
@@ -16,100 +17,174 @@ class ActivityHistory extends Component {
-              vector.push({
-                user: entry.user_edit_hash,
-                time: entry.timestamp_updated,
-                comment: 'change the status for ' + entry.status,
-              })
-            }
-            if (value.entries[index - 1].tags !== entry.tags) {
-              vector.push({
-                user: entry.user_edit_hash,
-                time: entry.timestamp_updated,
-                comment: 'change the tags for ' + entry.tags,
-              })
+          }
+          if (history.entries[index - 1].hierarchy !== entry.hierarchy) {
+            let icon = ''
+            if (entry.hierarchy == 'Leaf') {
+              icon = 'leaf.svg'
+            } else if (entry.hierarchy == 'Branch') {
+              icon = 'branch-with-leaf.png'
+            } else if (entry.hierarchy == 'Trunk') {
+              icon: 'trunk.png'
+            } else if (entry.hierarchy == 'Root') {
+              icon: 'root.png'
+            } else if (entry.hierarchy == 'No Hierarchy') {
+              icon: 'question-mark.svg'
             }
+            vector.push({
+              user: entry.user_edit_hash,
+              time: entry.timestamp_updated,
+              comment: `changed hierachy from "${
+                history.entries[index - 1].hierarchy
+              }" to "${entry.hierarchy}" `,
+              icon: icon,
+            })
+          }
+          if (history.entries[index - 1].description !== entry.description) {
+            vector.push({
+              user: entry.user_edit_hash,
+              time: entry.timestamp_updated,
+              comment: `changed description from "${
+                history.entries[index - 1].description
+              }" to "${entry.description}"`,
+              icon: 'font.svg',
+            })
+          }
+          if (history.entries[index - 1].status !== entry.status) {
+            vector.push({
+              user: entry.user_edit_hash,
+              time: entry.timestamp_updated,
+              comment: `changed status from "${
+                history.entries[index - 1].status
+              }" to "${entry.status}"`,
+              statusIcon: entry.status,
+            })
+          }
+          if (history.entries[index - 1].tags !== entry.tags) {
+            vector.push({
+              user: entry.user_edit_hash,
+              time: entry.timestamp_updated,
+              comment: 'change the tags for ' + entry.tags,
+            })
+          }
+        }
+      })
+      history.members.map(members => {
+        members.map((member, index) => {
+          if (index === 0) {
+            vector.push({
+              user: member.user_edit_hash,
+              time: member.unix_timestamp,
+              comment: `add "${
+                this.props.agents[member.agent_address].first_name
+              } ${
+                this.props.agents[member.agent_address].last_name
+              }"as a squirriel`,
```

Comment:
```suggestion
              }"as a squirrel`,
```
annoying word I know!